Configuration/Flexforms/Calendar.xml

```diff
@@ -40,6 +40,46 @@
                             </config>
                         </TCEforms>
                     </pages>
+                    <targetPid>
+                        <TCEforms>
+                            <exclude>1</exclude>
+                            <label>LLL:EXT:dlf/Resources/Private/Language/Calendar.xml:tt_content.pi_flexform.targetPid</label>
+                            <config>
+                                <type>group</type>
+                                <internal_type>db</internal_type>
+                                <allowed>pages</allowed>
+                                <size>1</size>
+                                <maxitems>1</maxitems>
+                                <minitems>0</minitems>
+                                <show_thumbs>1</show_thumbs>
+                                <wizards>
+                                    <suggest>
+                                        <type>suggest</type>
+                                    </suggest>
+                                </wizards>
+                            </config>
+                        </TCEforms>
+                    </targetPid>
+                    <initialDocument>
+                        <TCEforms>
+                            <exclude>1</exclude>
+                            <label>LLL:EXT:dlf/Resources/Private/Language/Calendar.xml:tt_content.pi_flexform.initialDocument</label>
+                            <config>
+                                <type>group</type>
+                                <internal_type>db</internal_type>
+                                <allowed>tx_dlf_documents</allowed>
+                                <size>1</size>
+                                <maxitems>1</maxitems>
+                                <minitems>0</minitems>
+                                <show_thumbs>1</show_thumbs>
+                                <wizards>
+                                    <suggest>
+                                        <type>suggest</type>
+                                    </suggest>
+                                </wizards>
+                            </config>
+                        </TCEforms>
+                    </initialDocument>
```

Comment:
I am not happy with this. Having just a single newspaper to present seems to be mostly an edge case. Most people don't need this and it can be achieved quite easily with pure typoscript as well:
`plugin.tx_dlf_calendar._DEFAULT_PI_VARS.id = [UID of document]`

I'd like to keep the flexform as clear as possible and suggest removing this feature. What do you think?

Reply:
I would provide both ways. The first one via flexform is more visible and an editor can use it too, to show the calendar somewhere in TYPO3. 
Additionally, I can overwrite it by a TypoScript setting which is bit more catchy with the other settings, we are using. E.g.:
```
plugin.tx_dlf_calendar {
    pages = {$config.storagePid}
    targetPid.data = TSFE:page|uid
    initialDocument = 10
    templateFile = {$config.templateFileCalendar}
}
```